src/main/java/Duke.java

```diff
@@ -1,3 +1,5 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
```

Comment:
I think the main method is too long and we can maybe refactor the code to make main shorter. According to our prof, it should "tell a story"